--- fs/mp/fordhmi/qml/Root.qml
+++ fs/mp/fordhmi/qml/Root.qml
@@ -407,9 +407,13 @@
         visible: active
                  && !ViewController.qrvc
                  && viewContainer.fullScreenViewActive === 0
                  && !root.fullScreenPopupActive
+                 && !root.hasActivePopup
+                 && !ViewController.transparentWindowPopupShown
+                 && !root.fullScreenTransparentPopupStacked
+                 && !(ViewManager.activeViewItem && ViewManager.activeViewItem.transparentWindow)
         source: active
                 ? "file:///fs/mp/fordhmi/qml/hmicustomapps/apps/Jan/Camper/Camper.qml"
                 : ""
